internal/donor/donorpage/progress.go

```diff
@@ -47,6 +47,18 @@ func Progress(tmpl template.Template, lpaStoreResolvingService LpaStoreResolving
 			Progress: progressTracker.Progress(lpa),
 		}
 
+		if !donor.WithdrawnAt.IsZero() {
+			data.InfoNotifications = append(data.InfoNotifications, progressNotification{
+				Heading: "lpaRevoked",
+				Body: appData.Localizer.Format(
+					"weContactedYouOnAboutLPARevokedOPGWillNotRegister",
+					map[string]any{"ContactedDate": appData.Localizer.FormatDate(donor.WithdrawnAt)},
+				),
+			})
+
+			return tmpl(w, data)
```

Comment:
I think we should still show any other relevant notifications

Reply:
but also I wanted to discuss on this ticket whether we should even do it, as the missing button seemed more "final"